### ISSUE TITLE
Load Camel K integrations from Java-file

### ISSUE THREAD

**Issue by NilsBerger** [citrusframework/yaks#319]
We currently start a camel k integration in a pre-script with the following command.

kamel run ../../../main/java/com/XXX/XXX/XXX/XXXXRouteBuilder.java \
  --wait -n name \
  --name camelk-test \
  --trait knative-service.min-scale=1 \
  --logging-level org.apache.camel=DEBUG \
  -d camel:camel-rest 

It would be nice if you could add a step that allows the setup of an integration with java file in the following way.

Given create Camel-K integration {name}.java with configuration:
  | dependencies | mvn:org.foo:foo:1.0,mvn:org.bar:bar:0.9 |
  | traits       | quarkus.native=true,quarkus.enabled=true,route.enabled=true |
  | properties   | foo.key=value,bar.key=value |

**Comment by christophd** [comment]
There actually is a step available:

```
  Scenario: Create integration with config
    When create Camel-K integration timertolog.groovy with configuration:
      | dependencies | mvn:fake.dependency:foo:1.0,mvn:fake.dependency:bar:0.9 |
      | traits       | quarkus.native=true,quarkus.enabled=true,route.enabled=true |
      | properties   | foo.key=value,bar.key=value |
      | source       | from('timer:tick?period=1000').setBody().constant('Hello world from Camel K').to('log:info') |
```

**Comment by christophd** [comment]
and you can load the file with Citrus function like this:

```
  Scenario: Create integration with config
    When create Camel-K integration timertolog.groovy with configuration:
      | dependencies | mvn:fake.dependency:foo:1.0,mvn:fake.dependency:bar:0.9 |
      | traits       | quarkus.native=true,quarkus.enabled=true,route.enabled=true |
      | properties   | foo.key=value,bar.key=value |
      | source       | citrus:readFile('classpath:path/to/route.java') |
```

**Comment by NilsBerger** [comment]
I get the following error when I try this out "Create integration with config (test.feature:2): Failure caused by com.consol.citrus.exceptions.TestCaseFailedException - Property  does not match format key=value". I use YAKS 0.4.0. 

Here is my code

---

yaks test --resource Class.java test.feature

---

Feature: Hallo
  Scenario: Create integration with config
    When create Camel-K integration timertolog.groovy with configuration:
       | dependencies | camel:camel-rest |
       | traits       | quarkus.native=true,quarkus.enabled=true,route.enabled=true |
       | name         | parquet-yaks-test |
       | source       | citrus:readFile('classpath:Class.java') |

---

public class Class {
    
}

**Comment by christophd** [comment]
I see your source code is Java but your integration indicates to use Groovy as you create the integration `timertolog.groovy`. Please try to create `timertolog.java` so you indicate to use Java as source code language for Camel-K.

Also could you please provide the full error StackTrace? I am not sure where this exception is raised and the StackTrace helps me to find the code throwing the exception. Thx

**Comment by christophd** [comment]
should be fixed with #337
